### ISSUE TITLE
Reset original defaults with `update_geom_defaults()`

### ISSUE THREAD

**Issue by bwiernik** [tidyverse/ggplot2#4993]
Sometimes, as I'm interactively building a plot, I change geom default aesthetics with `update_geom_defaults()`, then change my mind and want to put them back. Currently, I don't see a way to do that other than by knowing what the original defaults were or restarting my R session. 

It would be really helpful to be able to do something like this to reset the `size` default to 1.5:

```r
update_geom_defaults("geom", list(size = NULL))
```

And/or like this to reset all aesthetics to their original defaults:

```r
update_geom_defaults("geom", NULL)
```

**Comment by clauswilke** [comment]
As far as I can tell, we just overwrite the settings for the geom, so afterwards we don't have them anymore either.

https://github.com/tidyverse/ggplot2/blob/759c63c2fd9e00ba3322c1b74b227f63c98d2e06/R/geom-defaults.r#L14-L19

At some point there was talk about saving the geom defaults in the theme, and that would be a great solution for this problem, but I think there hasn't been much development on the PR for a long time now.

**Comment by hadley** [comment]
One small improvement here would be for `update_geom_defaults()` to invisibly return the previous values. That's our standard convention for functions that modify global state.

**Comment by davidhodge931** [comment]
Related to this, could there be a way to easily reset _all_ geom defaults back to originals? 

Maybe when the geom argument is NULL? 
`update_geom_updates(geom = NULL, update = NULL)`

Or maybe a new function like `reset_geom_updates()`?

**Comment by teunbrand** [comment]
Considering this closed (again) after #5846.
